src/main/java/mate/controller/RegisterCarController.java

```diff
@@ -0,0 +1,41 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/register")
```

Comment:
it would be better to change creational controller URLs as samples in the common mistakes
<img width="493" alt="image" src="https://user-images.githubusercontent.com/61326568/191191400-05727807-339c-4c4d-85e8-54b5de57a58c.png">

```suggestion
@WebServlet(urlPatterns = "/cars/register")
```